GameServerLib/GameObjects/Other/CollisionHandler.cs

```diff
@@ -5,14 +5,17 @@
 using LeagueSandbox.GameServer.GameObjects.AttackableUnits.AI;
 using LeagueSandbox.GameServer.GameObjects.AttackableUnits.Buildings.AnimatedBuildings;
 using LeagueSandbox.GameServer.Maps;
+using UltimateQuadTree;
 
 namespace LeagueSandbox.GameServer.GameObjects.Other
 {
     public class CollisionHandler: ICollisionHandler
     {
-        private Game _game;
+        private Game _game; //TODO: get rid of this thing YAGNI!
 
         private readonly List<IGameObject> _objects = new List<IGameObject>();
+        // This is the 'static map'
+        private readonly QuadTree<IGameObject> _quadMap = new QuadTree<IGameObject>(20000,20000,new CollisonObject()); //TODO: initialize the proper values of width and height for every map
```

Comment:
And what is the point of this static map? aside from addition and removal to it its not used.

Reply:
For future use. Check if champion collide with static objects and so, and we will have the option to reduce the number of checks even more.